src/app/cli/src/explore/api_server.rs

```diff
@@ -229,12 +230,94 @@ async fn root() -> impl axum::response::IntoResponse {
         <h1>Kamu HTTP Server</h1>
         <ul>
             <li><a href="/graphql">GraphQL Playground</li>
-            <li><a href="/swagger/">Swagger UI</li>
+            <li><a href="/openapi">OpenAPI Playground</li>
         </ul>
         "#
     ))
 }
 
+async fn openapi_playground() -> impl axum::response::IntoResponse {
+    // NOTE: Keep in sync with kamu-docs repo
+    axum::response::Html(indoc!(
+        r#"
```

Comment:
Minor: extract to e.g. `openapi-playground.html` and `include!()` here

Reply:
Great idea, thanks!